lib/reversetunnel/transport.go

```diff
@@ -91,7 +92,7 @@ func (t *TunnelAuthDialer) DialContext(ctx context.Context, _, _ string) (net.Co
 	}
 
 	// Check if t.ProxyAddr is ProxyWebPort and remote Proxy supports TLS ALPNSNIListener.
-	resp, err := webclient.Find(ctx, addr.Addr, t.InsecureSkipTLSVerify, nil)
+	resp, err := webclient.Find(ctx, addr.Addr, lib.IsInsecureDevMode(), nil, map[string]string{})
```

Comment:
I'm not that familiar with this codebase yet, so I just want to make sure that the change from `t.InsecureSkipTLSVerify` to `lib.IsInsecureDevMode()` is intentional, because it seems like an outlier to me.

In other callsites we just add `map[string]string{}` but here we also change one of the existing arguments.

Reply:
Yes, this change should probably be reverted.